client/galaxy/style/less/ui.less

```diff
@@ -152,7 +152,7 @@
             font-size: 0.8em;
             font-weight: normal;
             top: -1px;
-            word-break:break-word;
```

Comment:
I think this had no effect before as `break-word` seems to not be a valid value (`word-break: normal|break-all|keep-all|initial|inherit;` http://www.w3schools.com/cssref/css3_pr_word-break.asp)